website/content/docs/configuration/adaptive-overload-protection.mdx

```diff
@@ -0,0 +1,41 @@
+---
+layout: docs
+page_title: Adaptive Overload Protection - Configuration
+description: |-
+  Vault Enterprise provides Adaptive Overload Protection to automatically
+  prevent workloads from overloading different resources of the Vault servers.
+---
+
+# `adaptive_overload_protection`
+
+@include 'alerts/enterprise-only.mdx'
+
+@include 'alerts/beta.mdx'
+
+The `adaptive_overload_protection` stanza allows operators to control overload
+protection features. Changes to this stanza require a server restart to take
+effect.
+
+For more information read [Adaptive Overload
+Protection](/vault/docs/concepts/adaptive-overload-protection).
+
+
+```hcl
+adaptive_overload_protection {
+  disable_write_controller = false
+}
+```
+
+## `adaptive_overload_protection` parameters
+
+These parameters apply to the `adaptive_overload_protection` stanza in the Vault
+configuration file:
```

Comment:
```suggestion
```
It's obvious that the parameters are for `adaptive_overload_protection` given the section heading, the page title, and the doc intro. No need to repeat the introduction here.